Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Image/ImageQuality.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/base.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        using ImageQuality = u32;
```

Comment:
8bits would be more than enough, as this value won't really go over like 4-5. I put u32 as it's what's used to number the mipchains but it can be changed without any issue.